units/systemd-confext.service

```diff
@@ -26,9 +26,9 @@ Before=shutdown.target initrd-switch-root.target
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=systemd-confext refresh
-ExecReload=systemd-confext refresh
-ExecStop=systemd-confext unmerge
+ExecStart=@systemd-sysext systemd-confext refresh
+ExecReload=@systemd-sysext systemd-confext refresh
+ExecStop=@systemd-sysext systemd-confext unmerge
```

Comment:
Especially for this: systemd-confext is a public interface. It would be severely broken if the symlink doesn't exist...

Reply:
In this case, I do not have any strong opinion about the change. I will drop this if no one like it.